_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -10,4 +10,6 @@ This section will be updated prior to the next release.
 
 ### Upgrade warnings
 
+* Administer - Settings - Provisioning - DNS conflict timeout has been changed from integer to array of values (separated by comma). It used to represent timeout (in seconds) for DNS lookup attempts (dns_lookup macro and DNS record conflict validations). Multiple values in the array now represent multiple attempts to resolve name over DNS with the default value of [5, 10, 15, 20]. If this value was modified however, upgrade process will keep it. It's recommended to change it to perform multiple DNS query attempts.
```

Comment:
I realized the short name wasn't changed yet while reading this. The suggestion assumes https://github.com/theforeman/foreman/pull/6954 is merged.

```suggestion
* The DNS conflict timeout setting (found under Provisioning) has been changed from an integer to an array of values (separated by comma). It's also renamed to `DNS timeout`. It used to represent timeout (in seconds) for DNS lookup attempts such as the dns_lookup macro and DNS record conflict validations. Multiple values in the array now represent multiple attempts to resolve name over DNS with the default value of [5, 10, 15, 20]. If this value was modified however, the upgrade process will keep the single value. This means there's no retry. Users who've changed this value should review the new value.
```